File: apps/spring-boot-api/src/main/java/org/bootstrapbugz/api/auth/jwt/event/email/JwtEmail.java
package org.bootstrapbugz.api.auth.jwt.event.email;

import org.bootstrapbugz.api.shared.email.service.EmailService;
import org.bootstrapbugz.api.user.model.User;
import org.springframework.core.env.Environment;

public interface JwtEmail {
  void sendEmail(EmailService emailService, Environment environment, User user, String token);
}

File: backend/spring-boot-kotlin/src/main/kotlin/org/bootstrapbugz/api/auth/jwt/event/email/JwtEmail.kt
package org.bootstrapbugz.api.auth.jwt.event.email

import org.bootstrapbugz.api.shared.email.service.EmailService
import org.bootstrapbugz.api.user.model.User
import org.springframework.core.env.Environment

interface JwtEmail {
  fun sendEmail(emailService: EmailService, environment: Environment, user: User, token: String)
}
